src/core/unifiedtime.cpp

```diff
@@ -201,12 +201,12 @@ void ThreadUnifiedSamples(void* parg)
 		{
 		
 			/** Increment the Time Seed Connection Iterator. **/
-			nIterator++;
+			nIterator += GetRandInt(5);
 			
 			
 			/** Reset the ITerator if out of Seeds. **/
-			if(nIterator == SEEDS.size())
-			   nIterator = 0;
+			if(nIterator >= SEEDS.size())
+			   nIterator = GetRandInt(5);
```

Comment:
I don't think you want the "5" in here as that is the max rand number to generate.  I think what you want is to generate a random number up to as many seeds as there are, i.e.
nIterator = GetRandInt(SEEDS.size() - 1); // note the -1 as the nIterator is a 0-based index